src/schematools/types.py

```diff
@@ -475,12 +503,12 @@ def get_through_tables_by_id(self) -> List[DatasetTableSchema]:
         return tables
 
     @property
-    def display_field(self):
+    def display_field(self) -> Optional[Any]:
```

Comment:
Should this be `-> Optional[str]`

I am suggesting `str` as `display` is defined in the meta schema as:

```json
    "display": {
      "type": "string"
    },
```
However the docstring below suggest the result, if any, can be plural (fields). Should it be `Optional[List[str]]` then?

Reply:
It should indeed be `Optional[str]`. Changed.